docs/benchmark/allreduce/report.md

```diff
@@ -0,0 +1,101 @@
+# Benchmark: ElasticDL Calling the Fault-Tolerable AllReduce from Elastic Horovod
+
+We usually use synchronous distributed SGD to train deep learning models in CV
+and NLP. fault tolerance and Elastic scheduling can improve the resource utilization
+and user experience. Horovod has released fault-tolerance AllReduce in v0.20.0.
+ElasticDL implements elastic synchronous training on Kubernetes by calling
+Horovod. So, we experiment to verify the performance of ElasticDL.
+
+## Experiment 1: Elastic Scheduling of Multiple ElasticDL Jobs
+
+Supposed the GPU requirement of two jobs is a little over the total of the
+cluster. Without elastic scheduling, the latter job cannot start until
+another job completes. There are two problems:
+
+- Bad user experience. The submitter of the latter job must wait a long time.
+- Low resource utilization. The rest of GPU in the cluster is free.
+
+Using elastic scheduling, the latter job can start using the rest of GPU and
+does not need to wait for the completion of the earlier job.
+After the earlier job completes, the latter job can scale up to speed up
```

Comment:
scale up -> scale out
We have scale out/in for horizontal scaling which increase/decrease the instance count.
We have scale up/down for vertical scaling which increase/decrease the cpu/mem setting for each instance.